src/main/java/duke/Event.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Event extends Task{
+
+    /** Start date of Event */
+    protected String start;
+    /** End date of Event */
+    protected String end;
+    
+    public Event(String description, String start, String end) throws MissingParamsException{
+        super(description);
+        this.start = start;
+        this.end = end;
+
+        if(hasMissingParams()) {
```

Comment:
```suggestion
        if (hasMissingParams()) {
```
please check other areas of your code for missing spaces